.github/workflows/pre-release.yml

```diff
@@ -0,0 +1,71 @@
+name: Pre-release upload
+
+on:
+  schedule:
+    # Runs at 00:01 every day.
+    - cron: "1 0 * * *"
+
+concurrency:
+  # Include `github.event_name` to avoid pushes to `main` and
+  # scheduled jobs canceling one another
+  group: ${{ github.event_name }}-${{ github.ref }}
+  cancel-in-progress: true
+
+defaults:
+  # Required shell entrypoint to have properly activated conda environments
+  run:
+    shell: bash -l {0}
+
+jobs:
+  build:
+    name: Build and upload
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up python
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          miniforge-variant: Mambaforge
+          use-mamba: true
+          python-version: 3.9
+          environment-file: ci/environment.yml
+      - name: Install dependencies
+        run: |
+          mamba install boa conda-verify
```

Comment:
Is `conda-verify` needed?

Reply:
I believe it is if you want to run the verification parts while the build happens. We do have it in dask and distributed.